library_schemas/score/prerelease/HED_score_1.1.1.mediawiki

```diff
@@ -12,953 +12,587 @@ For more information see https://hed-schema-library.readthedocs.io/en/latest/ind
+            *** Rhythmic-trains-or-bursts-discharge-pattern <nowiki> {suggestedTag=Finding-prevalence, suggestedTag=Finding-frequency} [Applies to the intra-burst pattern: a non-ictal graphoelement that repeats itself without returning to the background activity between them. The graphoelements within this repetition occur at approximately constant period. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Arrhythmic-trains-or-bursts-discharge-pattern <nowiki> {suggestedTag=Finding-prevalence} [Applies to the intra-burst pattern: a non-ictal graphoelement that repeats itself without returning to the background activity between them. The graphoelements within this repetition occur at inconstant period. (Source: Beniczky ea 2017, Table 6;  Beniczky ea 2013, Appendix 4.)]</nowiki>
+            *** Fragmented-discharge-pattern <nowiki> [Source: Beniczky ea 2017, Table 6.] </nowiki>
 
-        ** Periodic-discharge-time-related-features <nowiki>[Periodic discharges not further specified (PDs) time-relayed features tags.]</nowiki>
-            *** Periodic-discharge-duration {suggestedTag=Property-not-possible-to-determine}
-                **** Very-brief-periodic-discharge-duration <nowiki>[Less than 10 sec.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Brief-periodic-discharge-duration <nowiki>[10 to 59 sec.]</nowiki>
-                    ***** <nowiki># {takesValue, valueClass=textClass}[Free text.]</nowiki>
-                **** Intermediate-periodic-discharge-duration <nowiki>[1 to 4.9 min.]</nowiki>
+        ** RPP-time-related-features <nowiki>[Time related features for Rhythmic or Periodic Patterns in critically ill patients (RPPs). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+            *** RPP-duration <nowiki> {suggestedTag=Property-not-possible-to-determine} [Duration (for PDs, RDA and SW). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** Very-brief-RPP-duration <nowiki>[Less than 10 sec. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** Brief-RPP-duration <nowiki>[10 to 59 sec. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** Intermediate-RPP-duration <nowiki>[1 to 4.9 min. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** Long-RPP-duration <nowiki>[5 to 59 min. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+                **** Very-long-RPP-duration <nowiki>[Greater than 1 hour. (Source: Beniczky ea 2017, Table 8.)]</nowiki>
+            *** RPP-onset <nowiki> {suggestedTag=Property-not-possible-to-determine} [Onset (for PDs, RDA and SW). (Source: Beniczky ea 2017, Table 8.)]</nowiki>
```

Comment:
Same comment as above about the suggestedTag

Reply:
fixed